Sources/Feather/Feather+DataBases.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  Feather+DataBases.swift
```

Comment:
Can move this logic back to the main file maybe? 

Honestly I don't feel that this should be separated from the main into an extension, it's just configuration.
If you really want to separate this logic, then we should create separate extensions for MySQL, Postgres and SQLite so that could be used in the main.swift file without the need of importing all the drivers, so people could easily choose between the drivers, without altering that much.

Reply:
The idea here was to use 1 docker supporting all 3 databases type. So the only way to do that is to import all of them.
Otherwise, we will need to have 1 docker per DB type.
 
So if we take the idea of separating in extension we will need 2 targets. Feather and Feather Docker maybe. Or we just create a repo specialised for Docker (which could be use for DockerHub as well). 

Let's discuss further this topic on Discord. (I have limited time today, so maybe this evening)

Reply:
Yeah I understand that we need to import all 3 db drivers, that's not a problem. 

I was thinking about smaller db config extensions in separate files that we can call in the main file. e.g.
ext1 -> Feather+configMySQL
ext2 -> Feather+configPostgres

then in the main file:
config: switch driver case "mysql" call.configMysql ; case "postgres" call.configPostgres

I also have time today, just DM me on discord so we can discuss this in detail. 👍